tests/unit/net/filemanager_thread.rs

```diff
@@ -5,18 +5,73 @@
 use ipc_channel::ipc::{self, IpcSender};
 use net::filemanager_thread::FileManagerThreadFactory;
 use net_traits::filemanager_thread::{FileManagerThreadMsg, FileManagerThreadError};
+use std::fs::File;
+use std::io::Read;
+use std::path::PathBuf;
 
 #[test]
 fn test_filemanager() {
     let chan: IpcSender<FileManagerThreadMsg> = FileManagerThreadFactory::new();
 
+    let opened = File::open("test.txt");
+    let mut test_file_content = vec![];
+    let opened_ok = opened.is_ok();
+
+    if let Ok(mut handler) = opened {
+        handler.read_to_end(&mut test_file_content)
+               .expect("Read tests/unit/net/test.txt error");
+    }
```

Comment:
Well, I made too loose an assumption before that the `test.txt` might not exist at all 😂